### ISSUE TITLE
A11y_Azure Tools for VS Code_Create New Project_Select a Language_Voiceover:When the voiceover focus moves to "Javascript' control inside 'Select a language', the listed item number visible on the speech viewer and is not being announced by the voiceover.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/vscode-azurefunctions#3157]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#BM_AzureToolsforVSCodeMAC_May2022;#Azure Tools for VS Code;#MAC;#FTP;#DesktopApp;#Voiceover;#WCAG1.3.1;#AINotInScope;#CEACCReviewed;#eDAD:3P:195086;#ThirdParty;#External:VoiceOver;#ExternalToApple;#Closed;

## Environment Details:

Application Name: Azure Tools for VS Code
Application Version: 1.66.2 (Universal)
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Visual Studio Code.
3. Click on extension button and search for "Azure Functions".
4. Click on install and tab to the "Azure" button.
5. Tab till 'Functions' button and hit enter.
6. Tab till 'Create new project' button and hit enter.
7. Tab till 'java Script' under 'Select a language'.
8. Use Voiceover to observe.

## Actual:
When the voiceover focus moves to "Javascript' control inside 'Select a language', the listed item number visible on the speech viewer and is not being announced by the voiceover.

The issue can be observed throughout the application.

## Expected:

When the voiceover focus moves to "Javascript' control inside 'Select a language', the listed item number visible on the speech viewer should be announced by the voiceover.

## User Impact:
Screen reader users may face difficulty in navigating in an organized manner.

## Attachment:
![NEW_A11y_Visual Studio Code for MAC_Create New Project_Select a Language_Voicoverr](https://user-images.githubusercontent.com/98393966/167087547-fec67395-63a5-4d68-889b-1ab9a8330fab.gif)

**Comment by MicroFish91** [comment]
Related upstream issue to follow is posted here: https://github.com/microsoft/vscode/issues/149395

**Comment by fsteffi** [comment]
external bug:https://github.com/microsoft/vscode/issues/149395

this bug should be in open state until external bug is fixed

**Comment by bbarik06** [comment]
External Bug to EDAD: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/195086
This bug should be in resolved state or open state (in GitHub) until the external bug is fixed and verified by testers.

**Comment by bbarik06** [comment]
@MicroFish91 Could you please provide the sample app or code pen. We have already logged the bug, but we need sample app for the external team so that they can produce a same control for them which isolates the problem and clarifies that the bug is in Apple's frameworks.

**Comment by MicroFish91** [comment]
Hi @bbarik06 unfortunately I'm not a code owner on that team, so I'm not sure where exactly in their repository to isolate that for you.  If I'm understanding the question correctly, I think it might be worth asking that question here:  https://github.com/microsoft/vscode/issues/149395

Thank you.

**Comment by bbarik06** [comment]
Asked for the details in the bug sent above https://github.com/microsoft/vscode/issues/149395

**Comment by fsteffi** [comment]
@isidorn  could you please provide us a codepen sample app for this app or issue as @MicroFish91  said above that he is not the owner of the code

the external team is waiting to the codepen to solve the issue
